### ISSUE TITLE
Local links on paths with special characters are not working

### ISSUE THREAD

**Issue by JanoschDeurer** [orzih/mkdocs-with-pdf#25]
There still seems to be a problem with links and special characters like ä,& ect. When creating a local link inside a markdown document like this:
```
[local link](#some-heading)
```
everything works fine as long as there is no special character in the path. With special characters in the path the link is no longer working in the pdf. However if we use the path explicitly like this:
```
[local link](../../../path/to/here_ä/#some-heading)
```
the link works again.

I created a minimal example to show the three different cases I described.
[mkdocs_heading_example.zip](https://github.com/orzih/mkdocs-with-pdf/files/5303783/mkdocs_heading_example.zip)

**Comment by orzih** [comment]
Thanks.
It fixed in v0.7.5.

**Comment by JanoschDeurer** [comment]
Thanks, this solves the last issue we had :)
